strax/context.py

```diff
@@ -474,7 +480,18 @@ def get_plugin(data_kind):
                 # drop the parents config from the lineage
                 configs = {}
                 for q, v in p.config.items():
-                    if q + p.child_ends_with in p.takes_config:
+                    # Looping over all settings, q is either the option name of the 
```

Comment:
perhaps you can juse parent_option_name instead of q. Makes the code much more readable

Reply:
Sure I will change it. I just stick to the previous notation.